src/Nowruz/modules/layout/components/statusDropDown/index.tsx

```diff
@@ -29,44 +30,54 @@ export const StatusDropDown: React.FC<StatusDropDownProps> = (props) => {
   const handleOutsideClick = (e) => {
     if (newRef.current && !newRef.current.contains(e.target)) { if (open) { setOpen(!open) } }
   }
+  // const handleOutsideClick = (e) => {
```

Comment:
If we don't need this maybe we can clear this out?

Reply:
I defined this function to close the menu when we click on the page and now I deleted it.